cherrypy/test/test_encoding.py

```diff
@@ -50,6 +50,8 @@ def cookies_and_headers(self):
                 cherrypy.response.cookie['candy']['domain'] = 'cherrypy.org'
                 cherrypy.response.headers[
                     'Some-Header'] = 'My d\xc3\xb6g has fleas'
```

Comment:
Maybe add u-prefix here?

Reply:
Right - definitely doesn't make sense to be making editorial changes in unaffected code.